sale_multiple_invoice/wizard/sale_make_invoice_advance.py

```diff
@@ -82,8 +83,9 @@ def create_invoices(self, cr, uid, ids, context=None):
                         months=invoice_count)).strftime(DEFAULT_SERVER_DATE_FORMAT)
                     invoice_id = self._create_invoices(
                         cr, uid, inv_values, sale_id, context=context)
+                    reference = _(sale_ref + ' Invoice ' + str(invoice_count + 1) + ' of ' + str(wizard.invoice_qty))
```

Comment:
reference = _("sale_ref + ' Invoice ' + str(invoice_count + 1) + ' of ' + str(wizard.invoice_qty))
Cambiar por algo así:
reference = _("%s Invoice %i of %i") % (sale_ref, invoice_count + 1, wizard.invoice_qty)

Podes probar hacer
ref_temp = _("%s Invoice %i of %i")
y lo usas luego directamente poniendo
ref_temp % (sale_ref, invoice_count + 1, wizard.invoice_qty)
y
ref_temp % (sale_ref, wizard.invoice_qty, wizard.invoice_qty)